advanced_samples/Bulk_Upload/DotNet/BulkUploaderTest/UnitTest1.cs

```diff
@@ -0,0 +1,119 @@
+using Bulk_Uploader;
+using Newtonsoft.Json;
+using OSIsoft.Data;
+using OSIsoft.DataViews;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace BulkUploaderTest
+{
+    public class UnitTest1
+    {
+        public static Exception toThrow = null;
```

Comment:
I would delete the `success` bool, and instead use `toThrow == null`. They should always resolve to the same boolean value.